internal/interceptor/inbound.go

```diff
@@ -24,7 +24,9 @@
 package interceptor
 
 import (
+	"context"
```

Comment:
Are those new changes under `internal/interceptor/...` really needed? I don't see any usage in other places in the PR.